CHANGES.rst

```diff
@@ -213,9 +213,12 @@ astropy.wcs
 Other Changes and Additions
 ---------------------------
 
-- Nothing changed yet.
-
+astropy.stats
+^^^^^^^^^^^^^
 
+- A Cython implementation for `astropy.stats.kuiper_two` and a vectorized
+  implementation for `astropy.stats.kuiper_false_positive_probability` have
+  been added, speeding up both functions.
```

Comment:
Please append with a "[#8104]" at the end to satisfy bot. Thanks for your patience! (If in doubt, see other entries.)

Reply:
done